gramps_webapi/auth/const.py

```diff
@@ -56,6 +56,7 @@
         PERM_ADD_OBJ,
         PERM_DEL_OBJ,
     },
+    ROLE_CONTRIBUTOR: {PERM_EDIT_OWN_USER, PERM_VIEW_PRIVATE, PERM_ADD_OBJ,},
```

Comment:
Maybe it seems too fine grained, but it seems natural to me you would want two versions of `ROLE_CONTRIBUTOR` and `ROLE_EDITOR`, one that is _trusted_ with `PERM_VIEW_PRIVATE` and one that is not. 

The one that is not would be for those with whom you may be working on older generations but who you are not close enough to that you would be willing to share everything.

Reply:
I thought about that, the reason I didn't add it (so far) was that granting `AddObject` without granting `ViewPrivate` leads to some additional complications that have to be handled. For instance, should we allow such a user to create on object that is itself private? Then they would have no way to view this object after creation.

Not saying it's impossible, but I'd suggest opening an issue and tackling this separately.

Reply:
Yeah that makes sense to handle that separately as you say. It would be nice if a `ViewLiving` was available separately too, but I seem to recall there were issues with that so you decided it made more sense to mark all living people private.

Reply:
Yes, we discussed this issue [in the forum](https://gramps.discourse.group/t/gramps-web-api-first-release/1609/5?u=davidmstraub). Still not sure how to solve it.